frontend/src/pages/Roster.js

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import MaterialTable from "@material-table/core";
+import { Helmet } from "react-helmet";
+import { Grid, Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { useDispatch } from "react-redux";
+import { TableIcons } from "../components/Icons";
+import LoadingContainer from "../components/LoadingContainer";
+import PageContainer from "../components/PageContainer";
+import { openAlert } from "../actions";
+import { getUsers } from "../services/users";
+import { withAuthorization } from "../components/HOC";
+
+const useStyles = makeStyles((theme) => ({
+  grid: {
+    textAlign: "center",
+  },
+  dates: {
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+const styleButton = {
+  color: "inherit",
+  fontWeight: "900",
+  textDecoration: "underline",
+  textTransform: "none",
+  background: "transparent",
+};
+const deafultButton = {
+  color: "gray",
+  fontWeight: "900",
+  textTransform: "none",
+};
+
+const Roster = () => {
+  const classes = useStyles();
+  const [state, setState] = React.useState({
+    // Initial backend data
+    loading: true,
+    developers: [],
+    alumni: [],
+  });
+  const [selectedRow, setSelectedRow] = React.useState(null);
+  const [alumni, setAlumni] = React.useState(false);
```

Comment:
Can we add these into the state on lines 38 - 43, or is there a reason you wanted them to be separate?